examples/advanced/demo_ri_basic.py

```diff
@@ -0,0 +1,317 @@
+# incorporating the correction for the convolution performed through the operator :math:`G`,
+# and :math:`\epsilon \in \mathbb{C}^{m}` is a realization of some i.i.d. Gaussian random noise.
+#
+# This operator can be implemented with `TorchKbNUFFT <https://github.com/mmuckley/torchkbnufft>`_.
+# Below, we propose an implementation wrapping it in :class:`deepinv.physics.LinearPhysics`.
+# As such, operations like grad and prox are available.
+
+from deepinv.physics import LinearPhysics
+
+
+class MeasOpRI(LinearPhysics):
+    r"""
+    Radio Interferometry measurement operator.
+
+    Args:
+        img_size (tuple): Size of the target image.
+        samples_loc (torch.Tensor): Normalized sampling locations in the Fourier domain.
+        dataWeight (torch.Tensor): Data weighting for the measurements.
+        device (torch.device): Device where the operator is computed.
+    """
+
+    def __init__(
+        self,
+        img_size,
+        samples_loc,
+        dataWeight=torch.tensor(
+            [
+                1.0,
+            ]
+        ),
+        device=torch.device("cpu"),
+        **kwargs,
+    ):
+        super(MeasOpRI, self).__init__(**kwargs)
+
+        self.device = device
+        self.nufftObj = tkbn.KbNufft(im_size=img_size, numpoints=7, device=device)
+        self.adjnufftObj = tkbn.KbNufftAdjoint(
+            im_size=img_size, numpoints=7, device=device
+        )
+        self.samples_loc = samples_loc.to(device)
+        self.dataWeight = dataWeight.to(device=device)
+
+    def setWeight(self, w):
+        self.dataWeight = w.to(device=device)
+
+    def A(self, x):
+        return self.nufftObj(x.to(torch.complex64), self.samples_loc) * self.dataWeight
+
+    def A_adjoint(self, y):
+        return torch.real(self.adjnufftObj(y * self.dataWeight, self.samples_loc)).to(
+            torch.float
+        )
+
+
+# %%
+# Groundtruth image
+# ----------------------------------------------------------------------------------------
+# The following data is our groundtruth with the settings of Experiment II in `Aghabiglou et al. (2024) <https://arxiv.org/abs/2403.05452>`_.
+# The groundtruth data has been normalized in the [0, 1] range.
+# As usual in radio interferometric imaging, the data has high dynamic range,
+# i.e. the ratio between the faintest and highest emissions is higher than in traditional low-level vision task.
+# In the case of this particular image, this ratio is of ``5000``.
+# For this reason, unlike in other applications, we tend to visualize the logarithmic scale of the data instead of the data itself.
+
+image_gdth = load_np_url(get_image_dataset_url("3c353_gdth", file_type="npy"))
+image_gdth = torch.from_numpy(image_gdth).unsqueeze(0).unsqueeze(0)
+
+
+def to_logimage(im, rescale=False, dr=5000):
+    r"""
+    A function plotting the image in logarithmic scale with specified dynamic range
+    """
+    if rescale:
+        im = im - im.min()
+        im = im / im.max()
```

Comment:
Shouldn't the rescaling be to [0,1]? 
For that, it should be something like `im = (im - im.min()) / (im.max() - im.min())`

Reply:
Yes, the rescaling is to [0,1] and the two ways are equivalent.

Reply:
Ah yes, my bad thanks @ChaoTang0330 :)